crates/bitwarden-core/src/auth/auth_request.rs

```diff
@@ -91,7 +91,9 @@ pub(crate) fn approve_auth_request(
     client: &Client,
     public_key: String,
 ) -> Result<UnsignedSharedKey, ApproveAuthRequestError> {
-    let public_key = AsymmetricPublicCryptoKey::from_der(&STANDARD.decode(public_key)?)?;
+    let public_key = AsymmetricPublicCryptoKey::from_der(&SpkiPublicKeyBytes::from(
```

Comment:
I've noticed that we do `Bytes::from(STANDARD.decode())` very often, I wonder if it would be simpler to have a `Bytes::from_b64()` function for that. Not as part of this PR though, just a possible future improvement.